Improve texts and start fixing accessibilty (#1742)
botpress/v12: docs/docs/building-chatbots/memory-&-data-persistence/flow-memory.md

--------------------

In a conversation, you may want to ask questions to the user and remember their answers for later use. You may also want to access the values that your chatbot extracts.

## System Parameters

> When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see a tree of all tracked system parameters. Click the debugger button in the top right corner and it will open in the bottom panel by default.

Author: ```suggestion
When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see a tree of all tracked system parameters. Click the debugger iconin the top right corner and it will open in the bottom panel by default.
```

button or icon?

> When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see the tree of all tracked system parameters. To open the tree in the bottom panel (default), click the debugger icon in the top right corner.

Author: ```suggestion
When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see the tree of all system parameters that it tracks. To open the tree in the bottom panel (default), click the debugger icon in the top right corner.
 
Add a suggestion, <Ctrl+g>
Add heading textAdd bold text, <Ctrl+b>Add italic text, <Ctrl+i>
Add a quote, <Ctrl+Shift+.>Add code, <Ctrl+e>Add a link, <Ctrl+k>
Add a bulleted list, <Ctrl+Shift+8>Add a numbered list, <Ctrl+Shift+7>Add a task list, <Ctrl+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply

```

> When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you can display a tree of all tracked system parameters. To open the tree in the bottom panel (default), click the debugger icon in the top right corner.

Author: ```suggestion
When a user talks to a chatbot, Botpress tracks all variables and parameters associated with that chatbot as the chatbot transitions from one state to another. If you run the debugger, you will see the tree of all tracked system parameters. To open the tree in the bottom panel (default), click the debugger icon in the top right corner.
```
